doc/source/serve/tutorials/intel-habana-gaudi-inference.md

```diff
@@ -6,16 +6,28 @@ orphan: true
 
 [Habana Gaudi AI Processors (HPUs)](https://habana.ai) are AI hardware accelerators designed by Habana Labs. For more information, see [Gaudi Architecture](https://docs.habana.ai/en/latest/Gaudi_Overview/index.html) and [Gaudi Developer Docs](https://developer.habana.ai/).
 
-This tutorial shows how to deploy [Llama2-7b](https://huggingface.co/meta-llama/Llama-2-7b-chat-hf) using a single HPU:
+This tutorial has two examples:
 
-* How to load a model in an HPU
+1. Deployment of [Llama2-7b](https://huggingface.co/meta-llama/Llama-2-7b-chat-hf) using a single HPU:
 
-* How to perform generation on an HPU
+    * How to load a model in an HPU
 
-* How to enable HPU Graph optimizations
+    * How to perform generation on an HPU
+
+    * How to enable HPU Graph optimizations
+
+2. Deployment of [Llama2-70b](https://huggingface.co/meta-llama/Llama-2-70b-chat-hf) using multiple HPUs on a single node:
+
+    * How to initialize distributed backend
+
+    * How to load a sharded model onto DeepSpeed workers
+
+    * How to stream responses from DeepSpeed workers
 
 This tutorial helps you serve a large language model on HPUs.
 
+Welcome to try out [llm-on-ray](https://github.com/intel/llm-on-ray), where you can find more examples, and get them deployed in a simpler way.
```

Comment:
Could you move this to the end of the tutorial in a separate `Next Steps` section? You could say:

```
Check out [llm-on-ray](https://github.com/intel/llm-on-ray) for more ways to customize and deploy LLMs at scale.
```